src/lib/y2storage/autoinst_profile/drive_section.rb

```diff
@@ -204,7 +204,10 @@ def init_from_disk(disk)
         return false if disk.partitions.empty?
 
         @type = :CT_DISK
+        # FIXME: could anyone with knowledge leave a comment why s390 is special here?
```

Comment:
my wild guess is that it is caused that s390 disk names depends on order of activation so very very unstable.

Reply:
OK, found reason:
https://github.com/yast/yast-autoinstallation/commit/8d4fe6c6c8dc37b866d39f1f95b3156db7bbd8b3

so this bug:
https://bugzilla.suse.com/show_bug.cgi?id=591603

Reply:
Yes, that's also what Arvin told me.

Still, I think not letting `@device` become `nil` is not a bad idea anyway.